release-notes/con-relnotes-notable-features.adoc

```diff
@@ -31,6 +31,8 @@ For more information, see the link:{LinkAdminGuide}#con-rbac-overview_admin-rhdh
 
 The GitHub Organization catalog module for the new backend system switched to a new `GithubMultiOrgEntityProvider` provider, which allows ingesting data from multiple GitHub organizations. This new provider brings in changes to the default plugin configuration settings.
```

Comment:
Can we call out that this is a breaking change that customers will need to react to?  I think we may need to have a migration instructions

Reply:
I have added a cross-ref to the procedure that covers the upgrade and configuration part: https://access.redhat.com/documentation/en-us/red_hat_developer_hub/1.1/html-single/getting_started_with_red_hat_developer_hub/index#enabling-github-organization-member-discovery-in-red-hat-developer-hub